compiler/testSuite/14-intersection/list1-v.bal

```diff
@@ -0,0 +1,13 @@
+import ballerina/io;
```

Comment:
We are abbreviating intersection to intersect, so call this 14-intersect

Reply:
fixed